doc: update documentation home page (#120)
anbox-cloud/discourse-docs: index.md

> Anbox Cloud is a cloud platform that containerises mobile workloads using Android as a guest operating system.

Author: ```suggestion
Anbox Cloud makes it possible to run Android apps in the cloud.
```

Reviewer: I feel like we miss what `.. for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices` was meant to say which is important to tell.

Author: Added back.

> It provides a rich software stack that enables you to run Android applications in the cloud. Anbox Cloud manages the container and application life cycle while optimising for high density, performance and fast container boot times.

Reviewer 2: I wonder if one of these first two paragraphs should mention the relationship of Anbox Cloud to Anbox. Even a "...using open-source [Anbox](https://anbox.io) containerisation technology." at the end of one of the existing sentences would provide some useful orientation.

> It provides a rich software stack that enables you to run Android applications in the cloud. Anbox Cloud manages the container and application life cycle while optimising for high density, performance and fast container boot times.

Author: ```suggestion
Using container technology, Anbox Cloud manages the application life cycle while optimising for high density, performance and fast container boot times. It is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures. Deploying the solution is simple and straightforward, and it is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).
```

Reviewer: nice!

> Using container technology, Anbox Cloud is scalable from a single-machine installation that can run scores of single Android systems to an entire cluster of machines. Deploying the solution is simple and straightforward, and it is based on powerful and battle-proven software from Canonical like [LXD](https://linuxcontainers.org/) and [Juju](https://jujucharms.com/).

Reviewer 2: I would like this to address the question of _why it exists_ more pointedly. What problem does Anbox Cloud solve? What was difficult, expensive or impossible until it existed? What pain has it relieved?

(As it stands, this paragraph seems to me to belong with the previous one: it says what Anbox Cloud does, not what need it meets.)

Author: How about:
```suggestion
Other solutions usually use virtual machines to emulate Android systems. Anbox Cloud, however, uses system containers, which results in an isolation level similar to virtual machines with an overhead as low as process containers. Therefore, Anbox Cloud can provide at least twice the container density compared to Android emulation in virtual machines, and can serve up to 100 Android instances per server.
```

Reviewer: As we plan to add virtual machine support at some point in the future we should reword here to not speak against using VMs (as there is a use case to have VMs). Instead leads emphasize that we concentrate on using system containers with LXD to deliver the best performance/density while still providing security and strict isolation.

Author: Tried to make it a bit more general.

> Anbox Cloud can be used for different kinds of use cases, from delivering central apps from the cloud to high-performance streaming of graphics to desktop and mobile client devices. Its platform integration tools allow, for example, the integration of existing streaming solutions. Anbox Cloud supports x86 and Arm64 hardware, providing the same set of features for both architectures.

Reviewer 2: This paragraph also seems to describe what Anbox Cloud does, and belongs with the second paragraph.

Instead, I'd like it to name the users whom it will help. Whose life will be made better by having Anbox Cloud? What industry/kind of organisation/kind of person benefits inn powerful ways?

Author: How about:
```suggestion
You should consider using Anbox Cloud if you want to deliver mobile applications that don't depend on the capabilities of your user's devices and that offload compute, storage and energy-intensive applications from the end device to the cloud. Typical use cases include mobile game streaming services, corporate application streaming, application automation and Android application testing.
```

Reviewer: +1

* Direct rendering for Intel and AMD GPUs
* OpenGL ES 3.2 support

For detailed information, see the [Release notes](https://discourse.ubuntu.com/t/release-notes/17842).

## Project and community

> Anbox Cloud is a Canonical product.

Reviewer: Should we call out here that Anbox Cloud is not open source and has no link to the open source Anbox?

Author: Yes, for sure!
I added `It is closed source and has no link to the Anbox open source project.`
A bit unsure if we should add a link to Anbox - we have it in the glossary, but I think it would be misleading here on the start page.

> Anbox Cloud is a Canonical product. It is closed source and has no link to the Anbox open source project.

Reviewer 2: I'd suggest:

```
## Development, support and user community

Anbox Cloud is a Canonical product.
```

To say "It is closed source [btw: 'proprietary' would be better] and has no link to the Anbox open source project" seems somehow a bit negative. Personally, I would not be inclined to say it. However use your judgement about whether this clarification is required.

Author: Discussed offline with @evildmp - we should keep it in this section and not move it to the intro. But we should word it more positively - something like "Anbox Cloud grew out of the Anbox open-source project. Its code base is now completely independent."
Does that sound better?

- [Get support through Ubuntu Advantage](https://ubuntu.com/support)

> - [Join the Discourse forum](https://discourse.ubuntu.com/c/anbox-cloud/49)

Reviewer 2: "Join the Discourse forum for [reasons/value]."
